simphony/testing/utils.py

```diff
@@ -86,12 +93,18 @@ def create_points():
         Point((0.0, 1.0, 1.0))]
 
 
-def create_bonds(n=5, restrict=None):
+def create_bonds(n=5, restrict=None, particles=None):
     bond_list = []
     for i in xrange(n):
         data = create_data_container(restrict=restrict)
-        data[CUBA.VELOCITY] = i
-        ids = [uuid.uuid4() for x in xrange(n)]
+        # FIXME: This is not right since CUBA.VELOCITY might not be part of the
```

Comment:
We need to have a way to differentiate the data containers for the others and at the same time adapt to the fact that the CUBA key that we use should be one of the supported cuba, preferably it should be a predictable value. I have some ideas but I need a little more time to think about it. 

Let me know if you want to give it a go!